src/main/webapp/aboutus.html

```diff
@@ -19,13 +19,23 @@
   <head>
     <meta charset="UTF-8">
     <title>About Us</title>
+		<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  	<link href="https://fonts.googleapis.com/css?family=Pinyon+Script&display=swap" rel="stylesheet">
+  	<link rel="stylesheet" href="/css/translation.css">
+  	<link rel="stylesheet" href="/css/main.css">
+ 		<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDTKCvhH556SBJtCN0MbZuOIPJX5rjYhgY"></script>
+ 		<script src="/js/navigation-loader.js"></script>
+
     <link rel="stylesheet" href="/css/main.css">
-  </head>
-  <body>
+		<link href="https://fonts.googleapis.com/css?family=Pinyon+Script&display=swap" rel="stylesheet">
+  	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+		<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDTKCvhH556SBJtCN0MbZuOIPJX5rjYhgY"></script>
+		<script src="/js/navigation-loader.js"></script>
```

Comment:
this part is repeated and you should delete unused scripts and css file link.

Reply:
Oh shoot I forgot to delete that my bad. I was messing with the links and scripts to figure out why the font won't update

Reply:
Ah wait I got it to work, hang on I'll commit the changes!